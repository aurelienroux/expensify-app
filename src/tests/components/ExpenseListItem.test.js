import React from 'react';
import { shallow } from 'enzyme';
import expenses from '../fixtures/expenses';
import ExpenseListItem from '../../components/ExpenseListItem';

test('should render ExpenseList item with a specific expense data', () => {
  const wrapper = shallow(<ExpenseListItem {...expenses[1]} />);
  expect(wrapper).toMatchSnapshot();
});