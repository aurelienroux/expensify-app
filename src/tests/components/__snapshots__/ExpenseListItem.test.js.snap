// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseList item with a specific expense data 1`] = `
<div>
  <Link
    to="/edit/2"
  >
    <h3>
      Rent
    </h3>
  </Link>
  <p>
    $1,095.00
    -
    1969, December 27th
  </p>
  <hr />
</div>
`;
